src/nowsecure-upload-action.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2021-2022 NowSecure Inc.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+import * as core from "@actions/core";
+import fs from "fs";
+import { NowSecure } from "./nowsecure-client";
+import { outputToDependencies, outputToSarif } from "./utils/action-utils";
+import * as github from "@actions/github";
+
+async function run() {
+  try {
+    const apiUrl = core.getInput("api_url");
+    const labApiUrl = core.getInput("lab_api_url");
+    const labUrl = core.getInput("lab_url");
+    const platformToken = core.getInput("token");
+    const ns = new NowSecure(platformToken, apiUrl, labApiUrl);
+    const groupId = core.getInput("group_id");
+    const appFile = core.getInput("app_file");
+    const enableSarif = core.getBooleanInput("enable_sarif");
+    const enableDependencies = core.getBooleanInput("enable_dependencies");
+    const githubToken = core.getInput("github_token");
+    const githubCorrelator = core.getInput("github_correlator");
+    //const headstart = parseInt(core.getInput("headstart_ms"), 10); // do we need this??
```

Comment:
should we remove this comment since it seems we don't need it?

Reply:
I'm happy to remove it, and we aren't using the headstart right now.  I left it in there so that a reviewer might chime in and give some reason that it is / was at one point necessary.